redi-connect-front/src/components/molecules/FilterDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { Checkbox, Icon } from '../atoms'
+import classnames from 'classnames'
+import './FilterDropdown.scss'
+
+interface Props {
+  label: string
+  selected: string[]
+  items: {label: string, value: string}[]
+  onChange: (item: string) => void
+}
+
+const FilterDropdown = ({ items, selected, label, onChange }: Props) => {
+  const baseClass = 'filter-dropdown'
+  const filterDropdown = useRef<any>(null)
+  const [showDropdown, setShowDropdown] = useState(false)
+
+  const handleClick = (e: any) => {
+    if (filterDropdown && !filterDropdown.current.contains(e.target)) {
```

Comment:
We can use this instead:
`if (!filterDropdown?.current?.contains(e.target as Node)) {`

Reply:
sure thing, after your update now we are in heaven